applications/callflow/src/cf_util.erl

```diff
@@ -339,7 +339,16 @@ vm_count_by_owner(<<_/binary>> = AccountDb, <<_/binary>> = OwnerId) ->
 %%--------------------------------------------------------------------
 -spec alpha_to_dialpad(ne_binary()) -> ne_binary().
 alpha_to_dialpad(Value) ->
-    << <<(dialpad_digit(C))>> || <<C>> <= strip_nonalpha(wh_util:to_lower_binary(Value))>>.
+    Stripped = [strip_nonalpha(C) || <<C>> <= wh_util:to_lower_binary(Value)],
+
+    Process = fun(Char, Acc) ->
+	case Char of
+	    [] -> Acc;
+	    _  -> [dialpad_digit(Char)|Acc]
+	end
+    end,
+
+    list_to_binary(lists:foldr(Process, [], Stripped)).
```

Comment:
Just a note here: when the function being used in a fold/map/etc isn't enclosing values from the outer scope, go ahead and use a named function instead. So the foldr should be `lists:foldr(fun convert_to_dialpad/2, [], Stripped)` instead of the anonymous function you created. Easier for the reader to get the intent of the fold too.